HTML/tables.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Basics of HTML and CSS</title>
+    <link href="styles/minimal-table.css" rel="stylesheet" type="text/css">
+  </head>
+  <body>
+    <h1>Here is where I would put a table...if i just had one!</h1>
+    <table>
+      <tr>
+        <th>Hi, I'm your first cell</th>
```

Comment:
`th` is to be used inside of a `<thead>` and is not good to mix `th` with `td` in the same row